ci_server/test_GitHub_API_functions.py

```diff
@@ -0,0 +1,49 @@
+from GitHub_API_functions import create_commit_status
```

Comment:
The file test_GitHub_API_functions.py should ideally be moved to test_ci_server so that they are run automatically when test_runner is run. This means that you'll also have to adjust the import statements in this line. Something like from ci_server.GitHub_API_functions import create_commit_status should work